modules/api/pkg/ee/clusterbackup/schedule/handler.go

```diff
@@ -250,3 +266,17 @@ func DecodeDeleteClusterBackupScheduleReq(c context.Context, r *http.Request) (i
 	}
 	return req, nil
 }
+
+func isClusterbackupEnabled(ctx context.Context, settingsProvider provider.SettingsProvider) error {
```

Comment:
Instead of creating the same function in three different places can we please create it at a common place and share this?